src/components/ImgSlider/ImgSlider.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import ArrowSvg from '../../helper-components/ArrowSvg';
+import styles from './ImgSlider.module.css';
+
+const ImgSlider = ({ game }) => {
```

Comment:
Was the implementation of this component chat GPT driven? it feels like it a bit :)